src/modm/platform/gpio/common/module.lb

```diff
@@ -18,7 +18,10 @@ def prepare(module, options):
     if not options[":target"].has_driver("gpio"):
         return False
 
-    module.depends(":architecture:gpio", ":utils")
+    module.depends(
+        ":architecture:gpio",
+        ":utils",
+    )
```

Comment:
But why? There are a lot of other cases where it's just one line.

Reply:
Yes, but this is better ;-) And I only fixed this occasion when I was editing this file for NXP port.
This is common practice to make diffs (when adding or removing an item) very crisp and clear. Note the comma at the end of line 23 which will not change line 23 if something is added to line 24. Would you agree?

Reply:
Come on.